form/unit_testing.rst

```diff
@@ -227,14 +227,15 @@ a good opportunity to use them::
 
     class TestedTypeTest extends TypeTestCase
     {
-        /**
-         * @dataProvider getValidTestData
-         */
+       
```

Comment:
thanks @apfelbox , it is clear that you are right
thanks @xabbuh, I'm happy to correct a Symfony's documentation for the second time :+1: